package.json

```diff
@@ -17,9 +17,9 @@
     "build:web": "cross-env PLATFORM=web vue-cli-service build",
     "build-zip": "node scripts/build-zip.js",
     "serve": "cross-env PLATFORM=web vue-cli-service serve",
-    "watch": "npm run build:extension -- --watch",
+    "watch": "npm run build:extension -- --watch --mode production",
```

Comment:
Is this one that is going to enable the hot reload that we discussed to add in Vue3 migration?

Reply:
This fixes the "hot-reload" for the extension, which broke when we upgraded to webpack 5